src/main/java/brooklyn/entity/proxy/aws/ElbController.java

```diff
@@ -93,10 +93,7 @@
     
     // http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/elasticloadbalancing/model/HealthCheck.html#setTarget(java.lang.String)
     ConfigKey<String> HEALTH_CHECK_TARGET = ConfigKeys.newStringConfigKey(
-            "aws.elb.healthCheck.target", "Specifies the instance being checked", "${instanceProtocol}:${instancePort?c}/");
-    
-    ConfigKey<Boolean> HEALTH_CHECK_ENABLED = ConfigKeys.newBooleanConfigKey(
-            "aws.elb.healthCheck.enabled", "Whether to do health checks for the instances", true);
+            "aws.elb.healthCheck.target", "The instance being checked. The protocol is either TCP, HTTP, HTTPS, or SSL. The range of valid ports is one (1) through 65535.", "HTTP:8080/");
```

Comment:
yep, ill remove the ports part.

I had problems with the old expression, but ill take another look at it.

the reason i removed the enable flag is because a health check is always created. So even if you say you do not want a healthcheck, a default one will be created.